BuildToolsVersion.txt

```diff
@@ -1 +1 @@
-1.0.27-prerelease-01007-02
\ No newline at end of file
+1.0.27-prerelease-01008-01
```

Comment:
This is a known issue which was fixed in CoreFx.  I haven't done any explicit validation for this in CoreClr though.